RibonDAO/interface
# feat: add authentication buttons to unauthorized modal [PROD3-543]

# Description

- Add sing in buttons in unauthorized modal
- 
## Type of change

## How to test
- Donate a ticket with magic link unauthorized email and check modal return contains sign in buttons (Gooogle, Appple, and Email).
